cpanfile

```diff
@@ -56,6 +56,7 @@ requires 'Devel::MAT::Dumper';
 
 # Things that may move out
 requires 'Term::ReadLine';
+requires 'Linux::Inotify2';
```

Comment:
This is platform-specific, we shouldn't have a hard requirement on it.

`recommends` or `suggests` perhaps?